client/src/components/application/Examine/ActionButtons.vue

```diff
@@ -322,10 +322,9 @@ export default {
 
         // initialize user in edit mode, with previous state set so NR gets set back to draft
         //  when user is done changing name, adding comment, etc.
-        this.$store.state.previousStateCd = 'DRAFT';
-        this.$store.state.is_editing = true;
+        this.$store.commit('setPreviousStateCd', 'DRAFT')
+        this.$store.commit('setEditing', true)
       }
-      this.$store.state.nrData.consent_dt = null
       this.$store.commit('resetConsent')
```

Comment:
`commit('resetConsent')` already sets `state.nrData.consent_dt` to `null`, so there is no need to seperately set it to null, which would require the creation of mutation anyway (which does not need to exist)